assets/js/modules/optimize/index.js

```diff
@@ -43,5 +44,21 @@ export const registerModule = ( modules ) => {
 			__( 'Improvement tracking', 'google-site-kit' ),
 			__( 'Probability and confidence calculations', 'google-site-kit' ),
 		],
+		checkRequirements() {
+			throw {
+				message: (
+					<div>
+						{ __(
+							'Optimize will no longer work after September 30, 2023.',
+							'google-site-kit'
+						) }{ ' ' }
+						<SupportLink path="/optimize/answer/12979939" external>
+							{ __( 'Learn more', 'google-site-kit' ) }
+						</SupportLink>
+					</div>
```

Comment:
This should use `createInterpolateElement` instead, so the whitespace is part of the translation string, eg.

```suggestion
					<div>
						{ createInterpolateElement(
							__(
								'Optimize will no longer work after September 30, 2023. <SupportLink>Learn more</SupportLink>',
								'google-site-kit'
							),
							SupportLink: <SupportLink path="/optimize/answer/12979939" external />
						) }
					</div>
```

Reply:
Thanks, @tofumatt. Updated.